### DISCUSSION TITLE
Pan Cam V2 Connection Timeout - Stand alone mode

### DISCUSSION THREAD

**Discussion by pjuhl2313** [gtxaspec/wz_mini_hacks#528] [Development]
Hey all,

I'm trying to get the mini_hacks installed on a V2 pan cam. I have firmware 4.49.1.47 installed. I've gotten the mini_hacks installed on 9 v3 cams without issue and per the install directions. The v2 pan cam should be the same install as the regular v3s. I add the SD card, hear Wyze mini, please wait. It gets a purple light, spins, then blinking red. BTW I'm installing this in stand alone mode. I hit the Ready to Connect button, scan the QR code for my Wifi and the connection times out. Any suggestions? Help?

**Comment by Bob-AOL** [comment]
The wi-fi configuration should already be established before booting wz_mini from the SD card. Unfortunately, 'Self-Hosted Mode' is only supported by v3 firmware 4.36.9.139, 4.61.0.1 and v2 firmware 4.9.8.1002 according to the Readme.
